linkis-computation-governance/linkis-engineconn/linkis-computation-engineconn/src/main/scala/org/apache/linkis/engineconn/computation/executor/bml/BmlEnginePreExecuteHook.scala

```diff
@@ -71,9 +72,9 @@ class BmlEnginePreExecuteHook extends ComputationExecutorHook with Logging {
               val fileName = resource.get(GovernanceConstant.TASK_RESOURCE_FILE_NAME_STR).toString
               val resourceId = resource.get(GovernanceConstant.TASK_RESOURCE_ID_STR).toString
               val version = resource.get(GovernanceConstant.TASK_RESOURCE_VERSION_STR).toString
-              val fullPath =
-                if (workDir.endsWith(seperator)) pathType + workDir + fileName
-                else pathType + workDir + seperator + fileName
+              val fullPath = fileName
+//                if (workDir.endsWith(seperator)) pathType + workDir + fileName
+//                else pathType + workDir + seperator + fileName
               val response = Utils.tryCatch {
                 bmlClient.downloadShareResource(processUser, resourceId, version, fullPath, true)
```

Comment:
why remove prefix path  for download?

Reply:
The replacementName already has the full address, so we need to cancel the splicing here